src/main/java/utils/DbUtils/Dao.java

```diff
@@ -24,18 +18,67 @@
         final String queryDelete = "DELETE FROM ADDRESS WHERE ID=7";
 
         // let us extract this and make a method
+        //Class.forName("oracle.jdbc.driver.OracleDriver");(Bunu kullanmama gerek var mi bilemedim, hata verip duruyor.
         Connection conn = DriverManager.getConnection(dbUri, userName, password);
-        Statement statement = conn.prepareStatement(queryCreate);
+        Statement statement = conn.createStatement();
+
+    public Dao() throws SQLException {
+    }
+
+
+    //ResultSet rs1 = statement.executeQuery(queryCreate);
+        //ResultSet rs2 = statement.executeQuery(queryUpdate);
+        //ResultSet rs3 = statement.executeQuery(queryDelete);
+
+                public void updateQuery () throws SQLException {
+                    try {
+                        statement.executeQuery(queryUpdate);
+                    } catch (SQLException e) {
+                        e.getErrorCode();
+                    }
+                }
+                public void createQuery () throws SQLException {
+                    try {
+                        statement.executeQuery(queryCreate);
+                    } catch (SQLException e) {
+                        e.getStackTrace();
+                    }
+                }
+                public void deleteQuery () throws SQLException {
+                    try {
```

Comment:
do you really need throws exception in the method signature?
since you are ctahcing these methods already?